test/src/test/java/hudson/model/UpdateSiteTest.java

```diff
@@ -132,6 +133,20 @@ public void shutdownWebserver() throws Exception {
         assertNotNull(us.getPlugin("AdaptivePlugin"));
     }
 
+    @Test
+    public void updateSiteWithCustomUpdateNowUpdatesData() throws IOException {
```

Comment:
Currently fails with

```
org.junit.ComparisonFailure: expected:<1.[2]> but was:<1.[1]>
	at org.junit.Assert.assertEquals(Assert.java:115)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at hudson.model.UpdateSiteTest.updateSiteWithCustomUpdateNowUpdatesData(UpdateSiteTest.java:147)
```

so it should be `@Ignore`d with a note to that effect.